man/man8/bashreadline.8

```diff
@@ -2,7 +2,7 @@
 .SH NAME
 bashreadline \- Print entered bash commands system wide. Uses Linux eBPF/bcc.
 .SH SYNOPSIS
-.B bashreadline
+.B bashreadline [\-h] [\-s SHARED]
```

Comment:
Looks like you added bashreadline related changes inadvertently. These changes are already merged. Could you remove them and resubmit?